hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -309,6 +310,8 @@ public AbfsRestOperation createFilesystem(TracingContext tracingContext)
 
     final AbfsUriQueryBuilder abfsUriQueryBuilder = new AbfsUriQueryBuilder();
     abfsUriQueryBuilder.addQuery(QUERY_PARAM_RESOURCE, FILESYSTEM);
+    // appending SAS Token to query
```

Comment:
nit: use "append" over "appending". or, given it is in the method name, cut entirely here and below

Reply:
Taken